model-loader-huggingface/src/test_utils.py

```diff
@@ -0,0 +1,19 @@
+import json
+import pytest
+from .utils import filter_files
+
+
+def test_filter_files():
+    files = ["pytorch_model.bin", "tf_model.h5"]
+    assert filter_files(files) == files[:1]
+
+    files = ["config.json",
+             "model-00001-of-00002.safetensors",
+             "model-00002-of-00002.safetensors",
+             "model.safetensors.index.json",
+             "pytorch_model-00001-of-00002.bin",
+             "pytorch_model-00002-of-00002.bin",
+             "pytorch_model.bin.index.json"]
+    assert filter_files(files) == [files[0], files[3], files[4], files[5], files[6]]
```

Comment:
Same tests just in a different form. `pytest.parametrize` is good for table-based testing:

```suggestion
@pytest.mark.parametrize(
    "files, expected",
    [
        (
            [
                "pytorch_model.bin",
                "tf_model.h5",
            ],
            ["pytorch_model.bin"],
        ),
        (
            [
                "config.json",
                "model-00001-of-00002.safetensors",
                "model-00002-of-00002.safetensors",
                "model.safetensors.index.json",
                "pytorch_model-00001-of-00002.bin",
                "pytorch_model-00002-of-00002.bin",
                "pytorch_model.bin.index.json",
            ],
            [
                "config.json",
                "model.safetensors.index.json",
                "pytorch_model-00001-of-00002.bin",
                "pytorch_model-00002-of-00002.bin",
                "pytorch_model.bin.index.json",
            ],
        ),
    ],
)
def test_filter_files(files, expected):
    assert filter_files(files) == expected
```

Reply:
I will adopt your test, looks cleaner and clearer for someone reading the test. Thanks for the suggestion!